.editorconfig

```diff
@@ -0,0 +1,11 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
```

Comment:
Now I state that it will be four with this project.
I think I just said that two spaces don't annoy me and that they were the default value, kind of.
Now we are using the same settings as other projects at Yseop.